mrchecker-framework-modules/mrchecker-database-module/pom.xml

```diff
@@ -88,6 +88,18 @@
             <version>${jpa.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.0.Final</version>
```

Comment:
please use variables, like ${hibernate-core.version}

Reply:
Like discussed, this functionality will be added using new pull-request.